src/hooks/useGetBooks.ts

```diff
@@ -0,0 +1,16 @@
+import { Items } from '../types/GoogleBooks'
+import { useGoogleBooksApi } from './useGoogleBooksApi'
+export const useGetBooks = (initialParameter: string) => {
+  const VOLUMES_SEAECH_PATH = '/volumes?q='
```

Comment:
normally shared constants would be defined outside the functions after the imports